model/src/main/java/org/cbioportal/genome_nexus/model/EnsemblTranscript.java

```diff
@@ -1,13 +1,13 @@
-package org.cbioportal.genome_nexus.annotation.domain;
+package org.cbioportal.genome_nexus.model;
 
-import org.cbioportal.genome_nexus.annotation.domain.internal.EnsemblRepositoryImpl;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 
 /**
  * Ensembl transcript
  */
-@Document(collection=EnsemblRepositoryImpl.TRANSCRIPTS_COLLECTION)
+//@Document(collection=EnsemblRepositoryImpl.TRANSCRIPTS_COLLECTION)
```

Comment:
Commented this out to prevent `model` module to depend on `persistence` module. We may want to keep these kind of shared constants in a different module rather than inside a specific `Impl` class.

Reply:
@onursumer yeah const module sounds good